roda-app.rb

```diff
@@ -70,6 +74,11 @@ class RodaApp < Roda
         r.route 'reservation'
       end
 
+      # /api/pull_requests
+      r.on "pull_requests" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.